documentation/ci-fix.md

```diff
@@ -137,6 +137,10 @@ We believe this is a false positive: https://github.com/Azure/azure-openapi-vali
 
 We believe this is a false positive: https://github.com/Azure/azure-openapi-validator/issues/642.  Until fixed, spec authors should **not** suppress the violations in `readme.md`, but rather have label `Approved-LintDiff` applied to their PR to ignore the errors.
 
+### `@singleton` causes `EvenSegmentedPathForPutOperation` and `XmsPageableForListCalls`
+
+If `EvenSegmentedPathForPutOperation` and/or `XmsPageableForListCalls` are failing for OpenAPI generated from TypeSpec using `@singleton` (OpenAPI path ends with `/default`), we believe this is a false positive: https://github.com/Azure/azure-openapi-validator/issues/646.  Until fixed, spec authors should **not** suppress the violations in `readme.md`, but rather have label `Approved-LintDiff` applied to their PR to ignore the errors.
```

Comment:
Suggestion:

Replace: 

> but rather have label `Approved-LintDiff` applied to their PR to ignore the errors.

With:

> but rather suppress the check per https://aka.ms/azsdk/pr-suppressions

Reply:
(to avoid duplication and teach the users about the docs they should know)

Reply:
I understand the goal to avoid duplicating the `Approved-LintDiff` label.  However, if we add this link from `ci-fix` to `pr-suppressions`, I think we are creating a loop.  Because in the `Suppression cases table` in page `pr-suppressions`, the value in column `PR suppression` for row `Swagger LintDiff` is a link back to `ci-fix`.

So a user goes to `ci-fix`, is told they need a PR-level suppression which is documented at `pr-suppressions`, but then the row for `Swagger LintDiff` directs them back to `ci-fix`?

I'm inclined to keep my original text rather than create a loop in our docs.

Reply:
Ok, cool. I plan to move this ci-fix file to azsdkdocs anyway, so I can fix this loop while I do that.